GOVERNANCE.md

```diff
@@ -0,0 +1,36 @@
+# Strimzi Governance
+
+This document defines project governance for the Strimzi project.
+
+## Voting
+
+The Strimzi project employs voting to ensure no single member can dominate the project. Any maintainer may cast a vote.
+
+For formal votes, a specific statement of what is being voted on should be added to the relevant github issue or PR.
+Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
+
+## Changes in Maintainership
+
+New maintainers are proposed by an existing maintainer and are elected by a ⅔ majority vote.
+
+Maintainers can be removed by a ⅔ majority vote.
+
+## Approving PRs
+
+PRs may be merged after receiving at least two positive votes. If the PR author is a maintainer, this counts as a vote.
+
+## Github Project Administration
+
+Maintainers will be added to the collaborators list of the Strimzi repository with "Write" access.
+
+After 6 months a maintainer will be given "Admin" access to the Strimzi GitHub organization.
+
+## Changes in Governance
+
+All changes in Governance require a ⅔ majority vote.
+
+## Other Changes
+
+Unless specified above, all other changes to the project require a ⅔ majority vote.
```

Comment:
I'm curious if there are also any plans to decentralize ownership of related Strimzi assets outside of GitHub control, such as domain names, twitter handles, and other project account information.  IIRC umbrella organizations like CNCF and Apache provide some kind of asset management here, though I can't find any such references.

Reply:
We should definitely do that. But I'm not sure whether that can be somehow covered in the governance policies unless we are actually accepted somewhere. Do you have any suggestion how to formulate it there?

Reply:
_For the record - the assets such as the domain name are currently privately owned._

Reply:
Adopting this governance model is just the first step on a path. The reality is that currently Strimzi is not currently part of any umbrella organization, so there is not yet any suitable legal entity to own such assets (and we're not going to create one just for Strimzi). We're exploring the possibility of becoming CNCF sandbox project, and adopting a neutral governance model is something they're looking for. If accepted then the maintainers would be guided by CNCF. If not then it would be up to the maintainers to decide how to proceed (and how quickly).

Reply:
> We're exploring the possibility of becoming CNCF sandbox project, and adopting a neutral governance model is something they're looking for. If accepted then the maintainers would be guided by CNCF. 

@tombentley I see.  I wasn't sure if this PR was created in conjunction with the application to CNCF or as something else.  I agree that they would likely provide guidance here.  In the past I've worked with [SPI](https://www.spi-inc.org/) and I know they provide this service.  I assume the same is true for other similar organizations.

> _For the record - the assets such as the domain name are currently privately owned._

@scholzj Seems reasonable to me.